style.css

```diff
@@ -463,23 +422,289 @@ header {
 }
 
 .submit-contact {
-  width: 131px;
-  margin-left: 98px;
-  margin-top: -12px;
+  width: 40.06%;
 }
 
 footer {
-  padding: 25px 95px;
+  width: 100%;
+  padding: 25px 0 5px 0;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
   border-top: 1px solid #dfe1e6;
   margin: 124px 0 8px 0;
+  box-sizing: border-box;
 }
 
 .footerIcon {
-  filter: invert(80%);
+  filter: invert(70%);
 }
 
-.headerLine {
+.footerLine {
   border-top: 5px solid #0d151e;
-  margin-left: 36px;
   width: 134px;
 }
+
+@media (min-width: 768px) {
+  /* header block */
+
+  .headTabs {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    column-gap: 32px;
+    justify-content: right;
+  }
+
+  .unionImg {
+    display: none;
+  }
+
+  header {
+    width: 100%;
+    padding-left: 7%;
+  }
+
+  main {
+    width: 100%;
+  }
+
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure that in the widescreen the page change smoothly without gaps
![Capture](https://user-images.githubusercontent.com/85095702/165848484-53d8d414-d5ee-407a-845e-fbd3a2cc83e1.PNG)

Reply:
@[phelian23](https://github.com/phelian23) I don't understand, when I look at it it is changing smoothly. This is really not ok. Neet to ask for an extra review but I don't know for what?

Reply:
@[phelian23](https://github.com/phelian23) As you can see I gave width 100%, I can't change properties on some devices. Can you give me at least a comment what I need to do?

Reply:
@[phelian23](https://github.com/phelian23) 
![image](https://user-images.githubusercontent.com/64913470/165854847-c3f2b902-e597-423e-80a5-e7202b540784.png). As you can see, it is working

Reply:
@Sanja969, Yes I can see that you try to fix it from the previous review, but the white space is still there just before it gets to full screen...I'll take a look at it and see what needs to change to correct it

Reply:
@Sanja969 Check the width of the `<div>` with class name `empty1` you gave it a fixed width, and that's what causing the issue. Kindly change the width to something like `80vw`

Reply:
@[phelian23](https://github.com/phelian23) Thank you, this helped me a lot. Please can you check again before I ask for an extra review, I don't want to repeat a week because of this stupid line. At least I will know in the future to use this unit. Maybe Github neet some time to update that demo link.

Reply:
@Sanja969, the line is okay now, also check the workproject class, you can give it a 95% width instead of the fixed-width you have, and then add the fixed-width as a max-width property

Reply:
I use vw instead of %, it works better, and gave max-width as you said. Please give me feedback if it is working  🙏

Reply:
It's okay now, the white space is no longer there